src/Symfony/Bundle/DebugBundle/Resources/config/services.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bridge\Monolog\Command\ServerLogCommand;
+use Symfony\Bridge\Twig\Extension\DumpExtension;
+use Symfony\Component\HttpKernel\DataCollector\DumpDataCollector;
+use Symfony\Component\HttpKernel\EventListener\DumpListener;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Command\Descriptor\CliDescriptor;
+use Symfony\Component\VarDumper\Command\Descriptor\HtmlDescriptor;
+use Symfony\Component\VarDumper\Command\ServerDumpCommand;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
+use Symfony\Component\VarDumper\Dumper\ContextProvider\CliContextProvider;
+use Symfony\Component\VarDumper\Dumper\ContextProvider\RequestContextProvider;
+use Symfony\Component\VarDumper\Dumper\ContextProvider\SourceContextProvider;
+use Symfony\Component\VarDumper\Dumper\ContextualizedDumper;
+use Symfony\Component\VarDumper\Dumper\HtmlDumper;
+use Symfony\Component\VarDumper\Server\Connection;
+use Symfony\Component\VarDumper\Server\DumpServer;
+
+return static function (ContainerConfigurator $container) {
+    $container->parameters()
+        ->set('env(VAR_DUMPER_SERVER)', '127.0.0.1:9912')
+    ;
+
+    $container->services()
+
+        ->defaults()
+            ->private()
```

Comment:
I added this line, because it was also configured in the xml file. But do we need to set all services to private explicitly when services are private by default?

Reply:
we don't need it. this was added in the XML file in the 3.x era, when this was not the default yet.